### DISCUSSION TITLE
Support labels in pull request templates

### DISCUSSION THREAD

**Discussion by MatiKloster** [community/community#12815] [Pull Requests]
I'd really like to have the ability to put labels in my `pull_request_template.md`.

We are using the GitHub integration on Slack. We've got a lot of channels subscribed to repository events with a specific label.
The problem is that the programmer must remember to add the corresponding label in order for the event to appear in the channel. It would be useful to create pull requests with a built-in label put so we don't have to remember to put a certain label all the time.

Something similar was discussed here: https://github.com/isaacs/github/issues/1252 but it was closed. Can we retake the conversation?

It would be amazing something like:
```md
labels:"won't fix"
---
PR template's description
```

Thanks

**Comment by MaxymVlasov** [comment]
Yeah, that feature request exist for 4 years...
https://github.com/isaacs/github/issues/1252

**Reply by MaxymVlasov** [reply]
For simple cases can be used https://github.com/github/issue-labeler.
But, in my case, PR creation trigger CI, which depends on PR labels.

If I need to change the default flow only for one repo, I can:
1. add a PR label automatically (Not possible now)
2. create a custom CI pipeline
3. create a GHA that will stop CI build, add a needed label, and start CI.

The first option, IMHO, Is preferable.

**Reply by ssbarnea** [reply]
isaccs is read-only since 2021, not really useful. Sadly I was not able to spot a fix inside templates.

**Comment by bskinn** [comment]
I don't know if an Actions-based workflow will work for all use-cases, but I ran across this recently, which might help sometimes: https://github.com/actions/labeler

**Comment by FredericVaugeoisFlo** [comment]
I need it to be automatic. It can't be in a workflow step. Also, it's possible to do it in an issues template, I don't know it's not possible to do it in the PR template...

**Comment by segraef** [comment]
Same, I wonder why it's working for issue templates but not pr templates.
